packages/i18n-utils/src/locales.ts

```diff
@@ -25,7 +25,7 @@ export const localesToSubdomains: Record< string, LocaleSubdomain > = {
 export const englishLocales: Locale[] = [ 'en', 'en-gb' ];
 
 // replaces config( 'livechat_support_locales' )
-export const livechatSupportLocales: Locale[] = [ 'en', 'es', 'pt-br' ];
+export const livechatSupportLocales: Locale[] = [ 'en' ];
```

Comment:
This isn't referenced in code anywhere... It looks like it was added in https://github.com/Automattic/wp-calypso/pull/46998 and the first commit on the PR actually uses this function, but then that reference was removed without removing the function definition.

Maybe we could just remove this line?

Reply:
Yeah, I wanted to do that, but the way I understand it, i18n-utils is a package that can be pulled in from other projects. I _doubt_ it's used by anyone (checked Automattic/happychat-client, just in case), but technically it could 🤔 
@lsl, as the author of that PR, could you weigh in here? Would be great if we could remove this duplication 🙇

Reply:
Apologies, missed this ping, this duplication is the result of some dependency juggling. Ideally we should be moving all of this out of config and into the i18n-utils package or something like it. Moving localizeUrl to a package needed the supporting config values moved with it (which I suspect also map directly to php config values.)